src/components/ui/Typography/Typography.stories.tsx

```diff
@@ -0,0 +1,35 @@
+import type { ComponentMeta, ComponentStory } from '@storybook/react';
+import Typography from './Typography';
+
+export default {
+  title: 'Ui/Typography',
+  component: Typography,
+  argTypes: {},
+} as ComponentMeta<typeof Typography>;
+
+const Template: ComponentStory<typeof Typography> = (args) => (
+  <Typography {...args}>
```

Comment:
We need a story for each entry in `classesMap` object + for each `weight`
Imagine that you're a developer who wants to understand how this component looks with different props and write stories according to that